### PR TITLE
Use PSQL/CRDB compatible query to fetch table information

### PR DESCRIPTION
**Pull request by cjfinnell**

### CODE REVIEW

File: tests/integration/table_details_test.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	crdbDefaultExpected := []db.Table{
+		{Name: "test1", Schema: "public", PKeyName: "primary", PKeyFields: map[int]string{1: "id1"}},
+		{Name: "test2", Schema: "public", PKeyName: "primary", PKeyFields: map[int]string{1: "id2"}},
+		{Name: "test3", Schema: "public", PKeyName: "primary", PKeyFields: map[int]string{1: "id3"}},
+	}
+	psqlDefaultExpectedTables := []db.Table{
+		{Name: "test1", Schema: "public", PKeyName: "test1_pkey", PKeyFields: map[int]string{1: "id1"}},
+		{Name: "test2", Schema: "public", PKeyName: "test2_pkey", PKeyFields: map[int]string{1: "id2"}},
+		{Name: "test3", Schema: "public", PKeyName: "test3_pkey", PKeyFields: map[int]string{1: "id3"}},
+	}
```
Comment (cjfinnell) [inline tests/integration/table_details_test.go]: The only difference between CRDB and PSQL here is that the name of the primary key constraint is different; PSQL uses `tablename_pkey`, while CRDB just uses `primary`

Comment (cjfinnell) [reply tests/integration/table_details_test.go]: I couldn't find anywhere in the codebase where we even reference `PKeyName` (might be able to deprecate the field) so I think it's a safe drift.

File: tests/integration/version_migrations_test.go [hunk 2]
```diff
@@ -101,9 +101,23 @@ func waitForPostgresReady(config *pgx.ConnConfig) bool {
...
+	cmd := []string{
+		"postgres",
+		"-cwal_level=logical",
+		"-cmax_replication_slots=1",
+		"-cmax_wal_senders=1",
+	}
+	return createDatabaseContainer(t, ctx, image, postgresPort, env, cmd)
+}
 
+func createDatabaseContainer(t *testing.T, ctx context.Context, image string, port int, env, cmd []string) (string, int, error) {
```
Comment (bernielomax) [inline tests/integration/version_migrations_test.go]: Shouldn't this just be `createContainer`?

File: tests/integration/table_details_test.go [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+)
+
+func TestGenerateTablesList(t *testing.T) {
+	// Re-used test case elements
+	defaultTableCreateQueries := []string{
+		`CREATE TABLE "test1" (id1 INT PRIMARY KEY, content TEXT);`,
+		`CREATE TABLE "test2" (id2 UUID PRIMARY KEY, content BYTEA);`,
+		`CREATE TABLE "test3" (id3 BIGINT PRIMARY KEY, content TIMESTAMP WITH TIME ZONE);`,
+	}
+	crdbDefaultExpected := []db.Table{
```
Comment (bernielomax) [inline tests/integration/table_details_test.go]: Maybe `crdbDefaultExpectedTables`?

**Review by bernielomax — APPROVED** [review]
This is pretty cool! Just a couple of nits.

File: tests/integration/version_migrations_test.go (hunk 2 above)
Comment (cjfinnell) [reply tests/integration/version_migrations_test.go]: changed in the rebase :+1:

File: tests/integration/table_details_test.go (hunk 3 above)
Comment (cjfinnell) [reply tests/integration/table_details_test.go]: changed in the rebase

**Pull request merged**